ui/src/time_picker.tsx

```diff
@@ -123,13 +142,13 @@ const
     <div data-test='lazyload' style={{ height: 59 }}>
       <Fluent.Spinner styles={{ root: { height: '100%' } }} size={Fluent.SpinnerSize.small} />
     </div>,
-  Toolbar = ({ setOpenView, time, label, switchAmPm }: ToolbarProps) =>
+  Toolbar = ({ setOpenView, time, label, switchAmPm, amPmDisabled }: ToolbarProps) =>
     <div className={css.toolbar}>
       {label && <Fluent.Label className={css.toolbarLabel}>{label}</Fluent.Label>}
       <Fluent.Text className={css.toolbarText}>
         <Fluent.Text className={css.toolbarTime} onClick={() => setOpenView('hours')}>{time?.substring(0, 2) || '--'}</Fluent.Text>:
         <Fluent.Text className={css.toolbarTime} onClick={() => setOpenView('minutes')}>{time?.substring(3, 5) || '--'}</Fluent.Text>{' '}
-        <Fluent.Text className={css.toolbarTime} onClick={switchAmPm}>{time?.substring(6, 8) || ''}</Fluent.Text>
+        <Fluent.Text className={clas(css.toolbarTime, amPmDisabled ? css.toolbarTextDisabled : css.toolbarTimeActive)} onClick={amPmDisabled ? undefined : switchAmPm}>{time?.substring(6, 8) || ''}</Fluent.Text>
```

Comment:
This is very long. Let's remove the whole styles + handler for switching ampm here. The native am/pm controls are enough.